src/Tizen.NUI/src/public/Rendering/RendererConstants.cs

```diff
@@ -0,0 +1,465 @@
+﻿// Copyright (c) 2019 Samsung Electronics Co., Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.ComponentModel;
+
+namespace Tizen.NUI
+{
+    /// <summary>
+    /// Enumeration for face culling mode.
+    /// </summary>
+    /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public enum FaceCullingModeType
+    {
+        /// <summary>
+        /// None of the faces should be culled
+        /// </summary>
+        NONE = 0,
+
+        /// <summary>
+        /// Cull front face, front faces should never be shown
+        /// </summary>
+        FRONT,
+
+        /// <summary>
+        /// Cull back face, back faces should never be shown
+        /// </summary>
+        BACK,
+
+        /// <summary>
+        /// Cull front and back faces; if the geometry is composed of triangles none of the faces will be shown
+        /// </summary>
+        FRONT_AND_BACK
+    }
+
+    /// <summary>
+    /// Enumeration for blend mode.
+    /// </summary>
+    /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public enum BlendModeType
+    {
+        /// <summary>
+        /// Blending is disabled.
+        /// </summary>
+        OFF = 0,
+
+        /// <summary>
+        /// Blending is enabled if there is alpha channel. This is the default mode.
+        /// </summary>
+        AUTO,
+
+        /// <summary>
+        /// Blending is enabled.
+        /// </summary>
+        ON,
+
+        /// <summary>
+        /// Blending is enabled, and don't cull the renderer.
+        /// </summary>
+        ON_WITHOUT_CULL,
+
+        /// <summary>
+        /// Blending is enabled when the actor is not opaque
+        /// </summary>
+        USE_ACTOR_OPACITY
+    }
+
+    /// <summary>
+    /// Enumeration for blend equation.
+    /// </summary>
+    /// This will be public opened in next tizen after ACR done. Before ACR, need to be hidden as inhouse API.
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public enum BlendEquationType
+    {
+        /// <summary>
+        /// The source and destination colors are added to each other.
+        /// </summary>
```

Comment:
Please add 'EditorBrowsable' tag here, too.

Reply:
This can be public open. In other words, The enums what I write `EditorBrowsable` in this scopes, are not ready to be public.